googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveEndpoint.java

```diff
@@ -72,7 +76,8 @@ public boolean isSingleton() {
     }
 
     Drive getClient(boolean forceFlag) {
-        if (client == null) {
+        if (client == null || forceFlag) {
+            LOG.info(" >> Get client");
```

Comment:
This line doesn't say that much. Either leave it away or provide more info what's going on:

 LOG.info("Get GoogleDrive client with clientid=" + );

Reply:
removed this log